frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx

```diff
@@ -0,0 +1,94 @@
+import * as classNames from 'classnames';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  YellowExclamationTriangleIcon,
+  GreenCheckCircleIcon,
+  BlueInfoCircleIcon,
+  RedExclamationCircleIcon,
+} from '@console/shared';
+import { ArrowCircleUpIcon } from '@patternfly/react-icons';
+import { global_info_color_100 as blueInfoColor } from '@patternfly/react-tokens';
+import { history, Timestamp } from '@console/internal/components/utils';
+
+import { NotificationTypes } from './notification-drawer';
+
+const NotificationIcon: React.FC<NotificationIconTypes> = ({ type }) => {
+  switch (type) {
+    case NotificationTypes.update:
+      return <ArrowCircleUpIcon color={blueInfoColor.value} />;
+    case NotificationTypes.success:
+      return <GreenCheckCircleIcon />;
+    case NotificationTypes.critical:
+      return <RedExclamationCircleIcon />;
+    case NotificationTypes.info:
+      return <BlueInfoCircleIcon />;
+    case NotificationTypes.warning:
+    default:
+      return <YellowExclamationTriangleIcon />;
```

Comment:
Should it default to `info` lvl?

Reply:
@benjaminapetersen , @cshinn  and I talked about defaulting to warning, but if you think info is better we can do info. This is the catch all for alerts where the notification type (which is a string for developers that create alerts) does not fall into one of the categories we provided. Let me know if you'd rather see info in this case.

Reply:
@cshinn my thought is that if someone mis-labeled or failed to label their warning, then it ought be considered unimportant.  Encourages devs to do the right thing :)

Reply:
@benjaminapetersen no objections, so changing this to default to info level alert when the severity is unknown.